packages/react-components/src/components/table-output-component.tsx

```diff
@@ -607,6 +607,7 @@ export class TableOutputComponent extends AbstractOutputComponent<TableOutputPro
                 if (currRowIndexFound && !isFound && rowNode.data && rowNode.data['isMatched']) {
                     this.gridApi?.ensureIndexVisible(rowNode.rowIndex);
                     this.selectStartIndex = this.selectEndIndex = rowNode.rowIndex;
+                    this.enableIndexSelection = true;
```

Comment:
I'm not sure this is needed. You can just check if selectStartIndex and selectEndIndex are set?